### ISSUE TITLE
bug: product ads homepage. 

### ISSUE THREAD

**Issue by gut1414** [revanced/revanced-patches#2349]
### Type

Error while patching

### Bug description

I seeing ads for product on YouTube homepage. 

### Steps to reproduce

I use latest updated youtube revanced..

### Relevant log output

```shell
I didn't found a way to give you a log.
```

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
![image](https://github.com/revanced/revanced-patches/assets/13122796/b42a69d3-ab29-4769-8fc8-cafbd6d84a15)
Report back

**Comment by ILoveOpenSourceApplications** [comment]
I'm also having this issue. Seems to be region specific. And just like the last time following these given instructions to capture the logcat isn't working out for me. And I forgot how I did it the last time. My system is only having the platform tools and nothing else. And grep isn't being recognised inside cmd.

**Comment by ILoveOpenSourceApplications** [comment]
> Replace grep with findstr and everything else should be the same.

`'logcat' is not recognized as an internal or external command, operable program or batch file.` this is the message I get when running the given command as it is.

**Comment by ILoveOpenSourceApplications** [comment]
adb logcat gives a long list from which I can seem to identify which is the one related to the issue. Though `expandable_product_grid` Seema to be related to it. But adding this to the custom filter doesn't remove everything.

**Comment by ILoveOpenSourceApplications** [comment]
`adb logcat | findstr AdsFilter | findstr Unblocked` is not showing anything output within cmd. Does that mean another command should be used inorder to save it as a text file?

**Comment by ILoveOpenSourceApplications** [comment]
Yes it is turned on.

**Comment by ILoveOpenSourceApplications** [comment]
Is `post_base_wrapper` a known YouTube element ID?

**Comment by oSumAtrIX** [comment]
It is not known to us, why?

**Comment by ILoveOpenSourceApplications** [comment]
I'm trying to find the component which is causing this issue, but I only found some of them and couldn't find for everything. These are the ones I found `offer_box,product_offers,expandable_product_grid,expandable_list_inner`. But still there are something things for which I can't identify the component name.

**Comment by ILoveOpenSourceApplications** [comment]
> Try just `adb logcat | findstr AdsFilter` and see what you get.
> 
> To save the output to a file use `adb logcat | findstr AdsFilter > log.txt`

This worked out for me. Though still going through the log to find which component caused the issue is still difficult. Also is it still `Unblocked` or was it changed to `Unfiltered`?

**Comment by ILoveOpenSourceApplications** [comment]
Caught the components. 
1) `expandable_metadata`
2) `expandable_list_inner`
Hiding these will remove the ads for products. 1 does this. 2 removes product from the description.

These ads are hidden when filtering using the above components but these are some of its sub components. Still posting here since I don't wanna capture a logcat anytime soon.
3) `styled_product_carousel`
4) `offer_box`
5) `product_offers`
6) `expandable_product_grid`
7) `shopping_flagging`.

Please do whatever necessary with these information.

**Comment by ILoveOpenSourceApplications** [comment]
Also, the command should be updated for all users, one for mobile, one for windows & one for Mac cause why not.

Windows command: `adb logcat | findstr AdsFilter | ads Unfiltered > log.txt`

**Comment by oSumAtrIX** [comment]
> 2 removes product from the description.

What do you mean by that?

> sub components

The child components do not matter. Once the parent component is filtered, child components will not be added and instead an `EmptyComponent` will be used, that effectively hides the ad.

**Comment by oSumAtrIX** [comment]
Have you confirmed the filter strings above don't cause any false positives? Use the app and check if it triggers anywhere, when the ad is not displayed.

**Comment by oSumAtrIX** [comment]
I also assume this, but I also assume that if we filter "too late" at some child component, we can't hide the outer parent, and that might leave some residue such as padding, requires testing

**Comment by ILoveOpenSourceApplications** [comment]
> > 2 removes product from the description.
> 
> What do you mean by that?

![Screenshot_20230611-113256~2](https://github.com/revanced/revanced-patches/assets/117499019/2a780d57-5040-4e3f-8d43-71e971bd1a48)

**Comment by ILoveOpenSourceApplications** [comment]
I'll provide the list here of what all is hidden by which components.

1) `inline_expander` can be used instead of `expandable_metadata`. It removes the below box entirely.
![Screenshot_20230611-115118~2](https://github.com/revanced/revanced-patches/assets/117499019/990a2819-eb5e-47eb-b4c5-158ef7ed2692)

2) `styled_product_carousel` hides the product images.
![Screenshot_20230611-120309~2](https://github.com/revanced/revanced-patches/assets/117499019/2aa3cbe3-b4f2-4cc3-a75c-ce5750c8061d)

3) `offer_box` hides the box which provides the offer price of the product.
Before:
![Screenshot_20230611-115125~3](https://github.com/revanced/revanced-patches/assets/117499019/35cf37cb-8526-4bbf-8b77-edd6a283f1fe)
After:
![Screenshot_20230611-120709~2](https://github.com/revanced/revanced-patches/assets/117499019/d52ed6d1-acd4-4d44-ab6a-0d483a2ba255)

4) `product_offers` hides the compare prices drop down and it's contents.
Before:
![Screenshot_20230611-120709~2](https://github.com/revanced/revanced-patches/assets/117499019/f0e77a26-a1fd-48fa-981f-13438562d26e)
![Screenshot_20230611-120605~3](https://github.com/revanced/revanced-patches/assets/117499019/dc149d6b-1741-45c3-8aac-d935f8d06abc)
After:
![Screenshot_20230609-230603~2](https://github.com/revanced/revanced-patches/assets/117499019/944b6f0b-f7a0-4484-8cf6-e3c5203ba129)

5) `expandable_product_grid` removes a long list of product ads which upon clicking takes user back to `offer_box`.
Before:
![Screenshot_20230611-121835~2](https://github.com/revanced/revanced-patches/assets/117499019/2b8b5fb3-7462-4737-b658-0843f17bf5c2)
![Screenshot_20230611-115209~2](https://github.com/revanced/revanced-patches/assets/117499019/0dfca460-515f-473e-a9ed-87c767c3d72d)
After:
![Screenshot_20230611-123043~2](https://github.com/revanced/revanced-patches/assets/117499019/4858af75-2cd2-4a1b-a535-e9c89b041a9a)

6) `shopping_flagging` removes the report listing option from `offer_box`
Before:
![Screenshot_20230611-115125~3](https://github.com/revanced/revanced-patches/assets/117499019/0ff8fa73-7e62-41f9-be25-cac9f7401f21)
After:
![Screenshot_20230611-123727~3](https://github.com/revanced/revanced-patches/assets/117499019/e8d5199f-4348-410a-a842-d308bb3f48d2)

**Comment by ILoveOpenSourceApplications** [comment]
![Screenshot_20230611-125646~2](https://github.com/revanced/revanced-patches/assets/117499019/7068aa8d-5284-4225-8cac-eccac106b274)

**Comment by ILoveOpenSourceApplications** [comment]
Yeah, some padding are left behind. 5 leaves behind the related products padding. Without 1 the banner stays. And adding 3, 4 & 6 leaves behind "Shop products in this video" padding and "Videos about this product".

**Comment by ILoveOpenSourceApplications** [comment]
When using logcat, I seem to be unavailable to capture the component names for these paddings and all. Is it because it's considered as another element? Is there any way using which I can identify these left overs?

**Comment by oSumAtrIX** [comment]
Left overs are caused by the parent elements. Litho components are recursively created and have child components. If you only hide a child component the parent will still be rendered. The most parent component is identified by the first name that is present in the pathBuilder string. Example:

a|b|c

Here c is a child of b which is a child of a where a may have a padding but you only hide b.

**Comment by ILoveOpenSourceApplications** [comment]
So `expandable_metadata` is the grandparent `inline_expansion` is the mother and `inline_expander` is the child if it's a|b|c. But there could be cases where there are a|b|c|d or more than 3 child components right?

But in this above case when using the parent and it's child the gap between two videos are extremely reduced when compared to using the grandchild. But the grandchild leaves behind extra gap between the two videos.

Using `expandable_metadata` & `inline_expansion`:
![Screenshot_20230615-191142](https://github.com/revanced/revanced-patches/assets/117499019/6461cdc1-0994-4abf-afbc-c7e414f17608)

 Using `inline_expander`:
![Screenshot_20230615-191428~2](https://github.com/revanced/revanced-patches/assets/117499019/4ff29575-78c3-45f3-9613-d455a49f7c63)

Original (without filter):
![Screenshot_20230615-191238~2](https://github.com/revanced/revanced-patches/assets/117499019/828feea7-b13e-4686-a7ce-f01e6711e7bb)

When a&b is being used the gap becomes non-existent and when using c the gap is slightly thicker than normal.

**Comment by oSumAtrIX** [comment]
Please send the full stringBuilder path

**Comment by ILoveOpenSourceApplications** [comment]
06-10 23:21:46.770 9674 9674 D revanced: AdsFilter: Exception (ID: null): related_video_with_context.eml/4bacb162c66cccb|related_video_with_context.enl 4bacb162c66cccb|CellType/active_view_display_container.eml|6f7403a6245fcf5c|ContainerType related_video_with_context_inner.eml |b4b978085a28a8f3|Container Type/expandable_metadata 85b3510ec09f3320|inline_expansion.eml/e25e2d7a520540bf|ContainerType|inline_expander.eml/3237dd8a1a2e4f22 | ContainerType|ContainerType

**Comment by oSumAtrIX** [comment]
Wait, is that really a slash

**Comment by ILoveOpenSourceApplications** [comment]
Don't mind the slash, I used Google Lens to copy the log from my laptop.

**Comment by oSumAtrIX** [comment]
Try active_view_display_container, should block it cleanly. Check if it also causes side effects

**Comment by ILoveOpenSourceApplications** [comment]
Now only shorts and community posts are available under the currently watching video when filtering this component  `active_view_display_container`

**Comment by oSumAtrIX** [comment]
What about related_video_with_context_inner

**Comment by ILoveOpenSourceApplications** [comment]
Does the same as the other component.

**Comment by oSumAtrIX** [comment]
Then the only way is via the children

**Comment by ILoveOpenSourceApplications** [comment]
Yeah, apparently `inline_expander` looks better than the others, combined with `expandable_list_inner` to remove the description product ads, everything is once again clean. The only side effect being a tiny bit more gap between adjacent videos which may go unnoticeable to the normal eye.

**Comment by ILoveOpenSourceApplications** [comment]
Will this be committed upon or is it going to be left here like this dude to these components possibility to being related to something else?

**Comment by oSumAtrIX** [comment]
Send a screenshot for every filter, before and after. The ones you sent are confusing.

**Comment by ILoveOpenSourceApplications** [comment]
> Using `expandable_metadata` & `inline_expansion`: ![Screenshot_20230615-191142](https://user-images.githubusercontent.com/117499019/246130187-6461cdc1-0994-4abf-afbc-c7e414f17608.png)
> 
> Using `inline_expander`: ![Screenshot_20230615-191428~2](https://user-images.githubusercontent.com/117499019/246130450-4ff29575-78c3-45f3-9613-d455a49f7c63.png)
> 
> Original (without filter): ![Screenshot_20230615-191238~2](https://user-images.githubusercontent.com/117499019/246130749-828feea7-b13e-4686-a7ce-f01e6711e7bb.png)

Are you talking about these before and after images?

**Comment by oSumAtrIX** [comment]
Yes, the images don't make sense. Please create a proper before and after.

**Comment by ILoveOpenSourceApplications** [comment]
In the first screenshot, the first video has the product container under it and while using the components `expandable_metadata/inline_expansion`, the gap between two videos is completely gone as the next video appears right below the channel name.
The second SC I'm holding down on the video, which produces a grey tint over the selected video showing how much space the video is taking and a black bar which isn't related anyways to the video. It's similar to the how `expandable_metadata/inline_expansion` works but there's an extra black space making it look like normal video gap is present between that video and the next one.
The third SC I'm holding down similarly, but the grey tint can be seen covering a bit more area below the channel name.

**Comment by oSumAtrIX** [comment]
Please do this

> Send a screenshot for every filter, before and after. The ones you sent are confusing.

**Comment by ILoveOpenSourceApplications** [comment]
Without any component filter:
![Screenshot_20230618-215407~2](https://github.com/revanced/revanced-patches/assets/117499019/defa7ce2-af21-42e1-ad44-8c1e3272e592)

With `expandable_metadata`:
![Screenshot_20230618-215544~2](https://github.com/revanced/revanced-patches/assets/117499019/d9d584fb-e876-4504-93a8-dcc755272fd8)

With `inline_expansion`:
![Screenshot_20230618-215725~2](https://github.com/revanced/revanced-patches/assets/117499019/547bf6b4-f561-41ed-91b6-843404d6ab4a)

With `inline_expander`:
![Screenshot_20230618-215921~2](https://github.com/revanced/revanced-patches/assets/117499019/8044f1b7-f320-4926-baf0-719187766ae7)

**Comment by oSumAtrIX** [comment]
Understood, so there is another type of `expandable_metadata` that YouTube uses to display chapters from videos. 

It looks like this but contains a list of chapters:

![image](https://github.com/revanced/revanced-patches/assets/13122796/6f34bd42-ac94-423a-90a5-e6aee679f1fc)

A patch for this already exists:

![image](https://github.com/revanced/revanced-patches/assets/13122796/b489ce8a-2bf4-4bc0-9d43-45bb1222a85f)

**Comment by oSumAtrIX** [comment]
Since this component is used for both showing products and chapters, hiding that component will hide both chapters and products under the video. I can rename the switch to "Hide expandable chip under videos", which would be the global term for both.

**Comment by oSumAtrIX** [comment]
What about `expandable_product_grid`, `offer_box` and the other components? When do they show up on YouTube?

**Comment by ILoveOpenSourceApplications** [comment]
> What about `expandable_product_grid`, `offer_box` and the other components? When do they show up on YouTube?

Only if the user is able to click on the product images which shows up in these expandable chips, at least that's how it is for now. I haven't come across any other situations leading to the 'shop items in this video' page yet.

**Comment by ILoveOpenSourceApplications** [comment]
Also using these components while filtering doesn't remove everything entirely as some elements are left over. The elements can be seen in this [comment](https://github.com/revanced/revanced-patches/issues/2349#issuecomment-1586051063).

**Comment by ILoveOpenSourceApplications** [comment]
> Understood, so there is another type of `expandable_metadata` that YouTube uses to display chapters from videos.

Sorry, didn't see this comment till now. Yes, YouTube does use a similar `expandable_metadata` container to list chapters under videos. And yes, it does look like this will hide that as well. But like I said it'll be better to use `inline_expander` instead as it provides a better look when hidden.


> A patch for this already exists:
> 
> ![image](https://user-images.githubusercontent.com/13122796/246677905-b489ce8a-2bf4-4bc0-9d43-45bb1222a85f.png)

Strange. Even though I had this option enabled, I was able to capture the element via logcat.

**Comment by oSumAtrIX** [comment]
Not sure; I don't see products, and I have this option on (default). Also `inline_expander` is a child of `expandable_metadata` so not sure why I would want to hide that instead of `expandable_metadata`

**Comment by ILoveOpenSourceApplications** [comment]
> Not sure; I don't see products, and I have this option on (default). Also `inline_expander` is a child of `expandable_metadata` so not sure why I would want to hide that instead of `expandable_metadata`

Better outcome? From this [comment](https://github.com/revanced/revanced-patches/issues/2349#issuecomment-1596198754) it can be seen that all three elements hides the chip entirely but the first two removed the gap between adjacent videos along with it while `inline_expander` only removes the chip and the gap, though slightly thicker, is still present.

**Comment by ILoveOpenSourceApplications** [comment]
Did the code for capturing log change?

> Windows command: `adb logcat | findstr AdsFilter | findstr Unfiltered > log.txt`

Cause this isn't working currently.

**Comment by oSumAtrIX** [comment]
Try LithoFilterPatch

**Comment by ILoveOpenSourceApplications** [comment]
That seems to work.

**Comment by ILoveOpenSourceApplications** [comment]
6-22 23:19:21.683 13655 25427 D revanced: LithoFilterPatch: Unfiltered (ID: cell_divider.eml | cbdd9c74ef2c193): cell_divider.eml cbdd9c74ef2c193 cell

ellTypel

6-22 23:19:21.683 13655 25427 D revanced: LithoFilterPatch: Filtered (ID: cell_divider.eml | cbdd9c74ef2c193): cell_divider.eml | cbdd9c74ef2c193 cell d

Why is cell_divider showing up under filtered an unfiltered.

**Comment by oSumAtrIX** [comment]
Please use codeblocks. The logs you sent make no sense.

**Comment by ILoveOpenSourceApplications** [comment]
I didn't copy and paste the entire log, just the part where the same component is being mentioned as filtered and unfiltered.

**Comment by oSumAtrIX** [comment]
![image](https://github.com/revanced/revanced-patches/assets/13122796/df25a162-e330-43ed-a09f-e60c24ab6d48)

Please usecodeblocks.

**Comment by ILoveOpenSourceApplications** [comment]
07-02 21:45:35.837 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: cell_divider.eml|cbdd9c74ef2c193): cell_divider.eml|cbdd9c74ef2c193|CellType|
07-02 21:45:35.837 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: cell_divider.eml|cbdd9c74ef2c193): cell_divider.eml|cbdd9c74ef2c193|CellType|

07-02 22:01:46.018 23273  7376 D revanced: LithoFilterPatch: Unfiltered (ID: cell_divider.eml|cbdd9c74ef2c193): cell_divider.eml|cbdd9c74ef2c193|CellType|
07-02 22:01:46.018 23273  7376 D revanced: LithoFilterPatch: Filtered (ID: cell_divider.eml|cbdd9c74ef2c193): cell_divider.eml|cbdd9c74ef2c193|CellType|

Yes, Hide Gray Separator is on.

07-02 21:45:35.819 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: null): expandable_list.eml|5f1b983e288bdd5a|expandable_list_inner.eml|d7cdda763cce72e6|CellType|ContainerType|ContainerType|product_link_item.eml|448417c243cf4498|ContainerType|ContainerType|product_image.eml|38741c470283edaa|ContainerType|ContainerType|
07-02 21:45:35.819 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: null): expandable_list.eml|5f1b983e288bdd5a|expandable_list_inner.eml|d7cdda763cce72e6|CellType|ContainerType|ContainerType|product_link_item.eml|448417c243cf4498|ContainerType|ContainerType|product_image.eml|38741c470283edaa|ContainerType|ContainerType|

Why was this not added within the ad filter?

07-02 21:45:34.887 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: products_in_video.eml|46339d899c0093e6): products_in_video.eml|46339d899c0093e6|ContainerType|
07-02 21:45:34.175 23273  7377 D revanced: LithoFilterPatch: Unfiltered (ID: null): offer_box_root.eml|a108848113b934da|CellType|offer_box.eml|c6ebe834d41a3ad5|ContainerType|ContainerType|ContainerType|TextType|
07-02 21:45:34.183 23273  7376 D revanced: LithoFilterPatch: Unfiltered (ID: product_details_unit.eml|669cf48a406254e1): product_details_unit.eml|669cf48a406254e1|product_details_unit.eml|669cf48a406254e1|CellType|

Or even though it is added it's still getting through?

**Comment by oSumAtrIX** [comment]
> Please use codeblocks. 

Please use codeblocks

**Comment by ILoveOpenSourceApplications** [comment]
Is it a separate software or code that I need to use?

**Comment by oSumAtrIX** [comment]
No, codeblocks 

```
Like
This
```

**Comment by ILoveOpenSourceApplications** [comment]
```java
07-02 22:01:46.018 23273 7376 D revanced: LithoFilterPatch: Unfiltered (ID: cell_divider.eml|cbdd9c74ef2c193): cell_divider.eml|cbdd9c74ef2c193|CellType|
07-02 22:01:46.018 23273 7376 D revanced: LithoFilterPatch: Filtered (ID: cell_divider.eml|cbdd9c74ef2c193): cell_divider.eml|cbdd9c74ef2c193|CellType|
07-02 21:45:35.819 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: null): expandable_list.eml|5f1b983e288bdd5a|expandable_list_inner.eml|d7cdda763cce72e6|CellType|ContainerType|ContainerType|product_link_item.eml|448417c243cf4498|ContainerType|ContainerType|product_image.eml|38741c470283edaa|ContainerType|ContainerType|
07-02 21:45:35.819 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: null): expandable_list.eml|5f1b983e288bdd5a|expandable_list_inner.eml|d7cdda763cce72e6|CellType|ContainerType|ContainerType|product_link_item.eml|448417c243cf4498|ContainerType|ContainerType|product_image.eml|38741c470283edaa|ContainerType|ContainerType|
07-02 21:45:34.887 23273 23273 D revanced: LithoFilterPatch: Unfiltered (ID: products_in_video.eml|46339d899c0093e6): products_in_video.eml|46339d899c0093e6|ContainerType|
07-02 21:45:34.175 23273 7377 D revanced: LithoFilterPatch: Unfiltered (ID: null): offer_box_root.eml|a108848113b934da|CellType|offer_box.eml|c6ebe834d41a3ad5|ContainerType|ContainerType|ContainerType|TextType|
07-02 21:45:34.183 23273 7376 D revanced: LithoFilterPatch: Unfiltered (ID: product_details_unit.eml|669cf48a406254e1): product_details_unit.eml|669cf48a406254e1|product_details_unit.eml|669cf48a406254e1|CellType|
07-02 22:09:31.527 23273  7376 D revanced: LithoFilterPatch: Unfiltered (ID: null): offer_box_root.eml|a108848113b934da|CellType|offer_box.eml|c6ebe834d41a3ad5|ContainerType|ContainerType|ContainerType|ContainerType|ContainerType|ContainerType|
07-02 22:09:31.527 23273  7377 D revanced: LithoFilterPatch: Unfiltered (ID: null): product_offers_root.eml|1ecf652f2a88d581|CellType|product_offers.eml|47e0f5dc28ada639|ContainerType|ContainerType|ContainerType|ImageType|
```

**Comment by ILoveOpenSourceApplications** [comment]
```java
07-02 22:09:37.111 23273  7377 D revanced: LithoFilterPatch: Unfiltered (ID: null): related_video_with_context.eml|1a205c0e26a397a0|CellType|active_view_display_container.eml|6f7403a6245fcf5c|ContainerType|related_video_with_context_inner.eml|d27a948eb95cb1c0|ContainerType|expandable_metadata.eml|2b91a52e39469622|inline_expansion.eml|2b0292cabd32bd7b|ContainerType|inline_expander.eml|1e8593184dc2e505|ContainerType|ContainerType|ContainerType|styled_product_carousel.eml|d000f2ab6a7f2434|CellType|horizontal_shelf.eml|7714f450376ddfd5|ContainerType|CollectionType|styled_product_carousel_item.eml|b6f66b32211ec7c0|CellType|ContainerType|ContainerType|ImageType|
```

**Comment by ILoveOpenSourceApplications** [comment]
So...what's going on here?

**Comment by ILoveOpenSourceApplications** [comment]
```java
07-02 23:01:05.847 16887 16887 D revanced: LithoFilterPatch: Unfiltered (ID: null): related_video_with_context.eml|18205c0e260397a0|CellType|active_view_display_container.eml|6f7403a6245fcf5c|ContainerType|related_video_with_context_inner.eml|d27a548eb95cb1c0 ContainerTypelslimline survey_wrapper.eml|1a8ae89b422a44a4| ContainerTypelslimline survey.eml|7cece179f10e692 ContainerType|ContainerType|smiley_button.eml|335 fed2a4566b068|ImageTypel
``` 

While at it, I was able to capture that survey component which popped up randomly below some videos as well.

**Comment by oSumAtrIX** [comment]
Survey's are blocked but there is a blacklist filter that prevents it from working.

**Comment by ILoveOpenSourceApplications** [comment]
Oh...so even if I add this component under custom filter you're saying that I may still see this survey?

**Comment by oSumAtrIX** [comment]
Custom filter bypass them

**Comment by ILoveOpenSourceApplications** [comment]
So it'll work then

**Comment by ILoveOpenSourceApplications** [comment]
Any idea why the hidden components are still showing up when capturing log?

**Comment by oSumAtrIX** [comment]
@ILoveOpenSourceApplications is this still an issue?

**Comment by ILoveOpenSourceApplications** [comment]
> @ILoveOpenSourceApplications is this still an issue?

Yes
> Hidden components show up with "Filtered" tag if hidden.

But some elements are not which can be seen in this [comment](https://github.com/revanced/revanced-patches/issues/2349#issuecomment-1616729409).

**Comment by oSumAtrIX** [comment]
If it says unfiltered, then it wasn't filtered

**Comment by oSumAtrIX** [comment]
Is this still a problem?
